CONTRIBUTING.md

```diff
@@ -3,6 +3,37 @@
 We'd love your help! Please join our weekly [SIG
 meeting](https://github.com/open-telemetry/community#special-interest-groups).
 
+## Target audiences
+
+The OpenTelemetry Collector has two main target audiences:
+
+1. End-users, aiming to use an OpenTelemetry Collector binary as a middleware on their infra, sitting between their
+workloads and the telemetry data storage.
+1. Downstream distributions, which can be an official one as part of the OpenTelemetry Collector project, such as the
```

Comment:
What do you think "Downstream" -> "Third Party"?

Reply:
I think downstream is appropriate here since even the first party distributions fall in this scope.  Currently the communication is simplified by the fact that the same group of people, by and large, are working both on the core framework and on the first party distributions, but that may not always be the case.

Reply:
Agree that third-party doesn't seem appropriate here because of the first party distributions. We use the words upstream / downstream within communications of the AWS distro among our team a lot but I don't know if that's common or standard - here it seems just the word "Distributions" without any qualifier is already clear enough if it isn't.

Reply:
Upstream/downstream is prevalent in other places as well, such as Linux distributions.